static/scss/shared-styles.scss

```diff
@@ -0,0 +1,13 @@
+img.rounded-profile-image {
+  &.small {
+    height: 43px;
+    border-radius: 50px;
+  }
+
+  &.medium {
+    height: auto !important;
```

Comment:
Gotcha. @roberthouse54, can you take a look at this? I've created #2732 to address this and other issues.